Client/Frontend/Browser/SavedTab+ConfigureExtension.swift

```diff
@@ -28,7 +28,17 @@ extension SavedTab {
             }
         }
         
-        self.init(screenshotUUID: tab.screenshotUUID, isSelected: isSelected, title: tab.title ?? tab.lastTitle, isPrivate: tab.isPrivate, faviconURL: tab.displayFavicon?.url, url: tab.url, sessionData: sessionData, uuid: tab.tabUUID, tabGroupData: tab.tabGroupData, createdAt: tab.firstCreatedTime, hasHomeScreenshot: tab.hasHomeScreenshot)
+        self.init(screenshotUUID: tab.screenshotUUID,
```

Comment:
So many params 🥲 Wondering if this could be at some point init with a struct or something

Reply:
it could be but I didn't work directly with SavedTab I just break down the parameters because the line was hard to read 🙂